middleware/authorization.js

```diff
@@ -56,32 +57,40 @@ function generateRefreshToken(rowInfo) {
 async function reIssueToken(req, res, next) {
     const accessToken = verifyToken(req.headers["authorization"], ACCESS_PRIVATE_KEY);
     const refreshToken = verifyToken(req.headers["refresh"], REFRESH_PRIVATE_KEY);
-
-    if (accessToken === "TokenExpiredError" && refreshToken === "TokenExpiredError") {
+    console.log("accessToken", accessToken);
+    console.log("refreshToken", refreshToken);
+    if ((accessToken === "TokenExpiredError") && (refreshToken === "TokenExpiredError")) {
 
         return res.redirect("/auth/login");
```

Comment:
```suggestion
        return res.status(401).json({message:"token expired"});
```

이 API를 호출하는 주체는 자바스크립트 단에서 이루어 집니다. 
그렇기때문에 리다이렉트는 정상적으로 처리 되지 않습니다.

그러므로 프론트에서 `refreshToken`이 만료되었음을 알려줘야 합니다.

만료응답을 받고, 프론트에서 로그인 페이지로 보낼겁니다.

물론 API문서에 해당내용이 포함되어있어야 합니다.